inlong-sort/sort-connectors/elasticsearch-7/src/main/java/org/apache/inlong/sort/elasticsearch7/table/Elasticsearch7Configuration.java

```diff
@@ -68,4 +69,38 @@ private static HttpHost validateAndParseHostsString(String host) {
                     e);
         }
     }
+
+    public List<HttpHost> getHosts() {
+        return config.get(HOSTS_OPTION).stream()
+                .map(Elasticsearch7Configuration::validateAndParseHostsString)
+                .collect(Collectors.toList());
+    }
+
+    @SuppressWarnings("unchecked")
+    public ActionRequestFailureHandler<DocWriteRequest<?>> getFailureHandler() {
+        final ActionRequestFailureHandler<DocWriteRequest<?>> failureHandler;
+        String value = config.get(FAILURE_HANDLER_OPTION);
+        switch (value.toUpperCase()) {
+            case "FAIL":
```

Comment:
The value name should be Failure handling strategy not Failure handler option  , and the value should be a enum type

Reply:
It is same as the origin handle in flink's elasticsearch connector.